src/components/players/detailsPlaceholder.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'src/scss/variables.scss';
+
+.playerDetailsContainer {
+    min-width: 400px;
+    padding: 10px;
```

Comment:
Looks better with `padding: 5px;`

Reply:
I noticed that in your sketch file it's padding 10px, but that in playersList container I only have it as 5px, so thats why it looks off. But I'd prefer to go off your sketch file dimensions, as it's easier to copy the values and more accurate to your design. So should I change the playersList container to padding; 10px instead?

Reply:
No, let's just make it padding: 5px, looks a lot better anyways. It's a minor thing, we can always adjust that padding later on.

Reply:
Let's leave it with `padding: 10px` and fix it later